transactions/update-contract/2022/aug-3/arguments.json

```diff
@@ -0,0 +1 @@
+[{"type":"String","value":"FlowToken"},{"type":"String","value":""}]
```

Comment:
@pgebheim Do you know what to use to generate the second argument here from the smart contract code?

Reply:
I don't!

I'm asking @janezpodhostnik  :-D

Reply:
The second argument should be the hex encoded contract. Its going to get used here: https://github.com/onflow/service-account/blob/main/templates/update_contract.cdc